docs/QSG/setup_centos.md

```diff
@@ -198,16 +196,14 @@ See
 <https://daos-stack.github.io/admin/deployment/#certificate-configuration>
 for more information.
 
-!!!note
-	The following commands are run from the `$ADMIN_NODE`.
+!!! note The following commands are run from the `$ADMIN_NODE`.
```

Comment:
Needs return and indent:
!!! note 
    The following commands are run from the `$ADMIN_NODE`.